morpheus-graphql-core/src/Data/Morpheus/Types/Internal/Validation/Validator.hs

```diff
@@ -281,7 +281,7 @@ fromValidationError
       scope = Scope {position, path}
     }
   err
-    | isInternal err || debug config =
+    | isInternal err || isCustom err || debug config =
```

Comment:
Please remove isCustom. Since we only want display internal context if error is unexpected or we are in debugging mode

Reply:
Done, I also added a test for this decoding in client.